AardvarkSample/AardvarkSample/AppDelegate.swift

```diff
@@ -1,8 +1,4 @@
 //
-//  AppDelegate.swift
-//  AardvarkSample
-//
-//  Created by Dan Federman on 9/21/16.
```

Comment:
I tend to keep this line, but I'm not terribly attached to it.

Reply:
To me it feels bit redundant, since that information should be available in the Git history. Any particular reason you think it's worth keeping?

Reply:
Gonna merge as-is to close this one out, but happy to revisit later if you want.